simpleregex/func.py

```diff
@@ -64,15 +64,15 @@ def any_of_characters(regex_list: List[str]):
 
 def times(what: RegEx, min: int, max: int = None):
     """
-    Repeate a pattern between {min} and {max} times.
+    Repeat pattern between {min} and {max} times.
```

Comment:
I would probably leave it as 
```suggestion
    Repeat a pattern between {min} and {max} times.
```
But anyways I believe it's alright 👍